govcd/vapp.go

```diff
@@ -136,10 +136,8 @@ func (vapp *VApp) AddVM(orgVdcNetworks []*types.OrgVDCNetwork, vappNetworkName s
 			},
 			InstantiationParams: &types.InstantiationParams{
 				NetworkConnectionSection: &types.NetworkConnectionSection{
-					Type:                          vappTemplate.VAppTemplate.Children.VM[0].NetworkConnectionSection.Type,
```

Comment:
Can this break functionality if someone depends on it using the SDK?

Reply:
Might break it for someone eventually. But it also fixes #213 and #26

Reply:
Ok. Fair enough. Could you put a line near some notes in changelog that it is changed and a short why. That gives someone perspective.